IntegrationAdapters/Repositories/ActionsAndBenefitsRepository.cs

```diff
@@ -0,0 +1,42 @@
+﻿using IntegrationAdapters.Models;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace IntegrationAdapters.Repositories
+{
+    public class ActionsAndBenefitsRepository
+    {
+        public readonly HealthCareSystemDbContext dbContext;
+
+        public ActionsAndBenefitsRepository(HealthCareSystemDbContext context)
+        {
+            this.dbContext = context;
```

Comment:
You should instantiate the context here, and not get it all the way from controller

Reply:
Other team members have correct that so we are going to see how they have done that :)